executors/stable/executor.py

```diff
@@ -0,0 +1,154 @@
+import argparse, os, sys, glob
+from io import BytesIO
+import torch
+import numpy as np
+from PIL import Image
+from tqdm import tqdm, trange
+from itertools import islice
+from einops import rearrange
+import time
+from pytorch_lightning import seed_everything
+from torch import autocast
+from contextlib import nullcontext
+from typing import Dict
+from random import randint
+
+from ldm.util import instantiate_from_config
+from ldm.models.diffusion.ddim import DDIMSampler
+from ldm.models.diffusion.plms import PLMSSampler
+
+from jina import Executor, DocumentArray, Document, requests
+from omegaconf import OmegaConf
+
+
+class StableDiffusionConfig():
+    '''
```

Comment:
this looks like it should be a dataclass

Reply:
Yes, should be a subclass of `Executor`

Reply:
Sure